### PR TITLE
test_in_tail: fix flaky tests for file rotation

### LINKED ISSUE #4707: CI: Flaky "test_next_rotation_occurs_very_fast_while_old_TW_still_waiting_rotate_wait" test
### Describe the bug

Sometimes, test_next_rotation_occurs_very_fast_while_old_TW_still_waiting_rotate_wait will be failed 

```
3) Failure: test_next_rotation_occurs_very_fast_while_old_TW_still_waiting_rotate_wait(TailInputTest::Update watchers for rotation without follow_inodes)
D:/a/fluentd/fluentd/test/plugin/test_in_tail.rb:3358:in `test_next_rotation_occurs_very_fast_while_old_TW_still_waiting_rotate_wait'
     3355:         end
     3356:       end
     3357: 
  => 3358:       assert_equal(
     3359:         {
     3360:           record_values: ["file1 log1", "file1 log2", "file2 log1", "file2 log2", "file3 log1", "file3 log2"],
     3361:           tail_watcher_paths: ["#{@tmp_dir}/tail.txt0", "#{@tmp_dir}/tail.txt0", "#{@tmp_dir}/tail.txt0"],
<{:position_entries=>
  [["D:/a/fluentd/fluentd/test/plugin/../tmp/tail/0f6319cbb74ae1cd5cae/tail.txt0",
    "0000000000000016",
    nil]],
 :record_values=>
  ["file1 log1",
   "file1 log2",
   "file2 log1",
   "file2 log2",
   "file3 log1",
   "file3 log2"],
 :tail_watcher_inodes=>[4728759779328, 4737349320704, nil],
 :tail_watcher_io_handler_opened_statuses=>[false, false, false],
 :tail_watcher_paths=>
  ["D:/a/fluentd/fluentd/test/plugin/../tmp/tail/0f6319cbb74ae1cd5cae/tail.txt0",
   "D:/a/fluentd/fluentd/test/plugin/../tmp/tail/0f6319cbb74ae1cd5cae/tail.txt0",
   "D:/a/fluentd/fluentd/test/plugin/../tmp/tail/0f6319cbb74ae1cd5cae/tail.txt0"]}> expected but was
<{:position_entries=>
  [["D:/a/fluentd/fluentd/test/plugin/../tmp/tail/0f6319cbb74ae1cd5cae/tail.txt0",
    "0000000000000016",
    4741644025856]],
 :record_values=>
  ["file1 log1",
   "file1 log2",
   "file2 log1",
   "file2 log2",
   "file3 log1",
   "file3 log2"],
 :tail_watcher_inodes=>[4728759779328, 4737349320704, nil],
 :tail_watcher_io_handler_opened_statuses=>[false, false, false],
 :tail_watcher_paths=>
  ["D:/a/fluentd/fluentd/test/plugin/../tmp/tail/0f6319cbb74ae1cd5cae/tail.txt0",
   "D:/a/fluentd/fluentd/test/plugin/../tmp/tail/0f6319cbb74ae1cd5cae/tail.txt0",
   "D:/a/fluentd/fluentd/test/plugin/../tmp/tail/0f6319cbb74ae1cd5cae/tail.txt0"]}>
diff:
  {:position_entries=>
    [["D:/a/fluentd/fluentd/test/plugin/../tmp/tail/0f6319cbb74ae1cd5cae/tail.txt0",
      "0000000000000016",
?     nil          ]],
?     4741644025856   
?     ???   
   :record_values=>
    ["file1 log1",
     "file1 log2",
     "file2 log1",
     "file2 log2",
     "file3 log1",
     "file3 log2"],
   :tail_watcher_inodes=>[4728759779328, 4737349320704, nil],
   :tail_watcher_io_handler_opened_statuses=>[false, false, false],
   :tail_watcher_paths=>
    ["D:/a/fluentd/fluentd/test/plugin/../tmp/tail/0f6319cbb74ae1cd5cae/tail.txt0",
     "D:/a/fluentd/fluentd/test/plugin/../tmp/tail/0f6319cbb74ae1cd5cae/tail.txt0",
     "D:/a/fluentd/fluentd/test/plugin/../tmp/tail/0f6319cbb74ae1cd5cae/tail.txt0"]}
```

### To Reproduce

Run CI

### Expected behavior

No errors

### Your Environment

```markdown
- Fluentd version:
- Package version:
- Operating system:
- Kernel version:
```

### Your Configuration

```apache
none
```

### Your Error Log

```shell
See above
```

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Watson1978**
**Which issue(s) this PR fixes**: 
Fixes #4707

**What this PR does / why we need it**: 
This PR will fix unstable tests.

**Docs Changes**:

**Release Note**:

### CODE REVIEW

File: test/plugin/test_in_tail.rb [hunk 1]
```diff
@@ -3265,6 +3265,7 @@ def test_refreshTW_during_rotation
       inode_0 = tail_watchers[0]&.ino
       inode_1 = tail_watchers[1]&.ino
       inode_2 = tail_watchers[2]&.ino
```
Comment (daipom) [inline test/plugin/test_in_tail.rb:3267]: Looks like we don't need to assert these inode values in this test.
Could you please remove them in this PR?

File: test/plugin/test_in_tail.rb [hunk 2]
```diff
@@ -3346,6 +3347,7 @@ def test_next_rotation_occurs_very_fast_while_old_TW_still_waiting_rotate_wait
...
```
Comment (daipom) [inline test/plugin/test_in_tail.rb:3349]: ditto.

**Review by daipom — COMMENTED** [review]
Thanks!! Great!!

File: test/plugin/test_in_tail.rb (hunk 1 above)
Comment (Watson1978) [reply test/plugin/test_in_tail.rb:3267]: OK, I will remove them.
Thanks

Comment (Watson1978) [reply test/plugin/test_in_tail.rb]: Fixed with https://github.com/fluent/fluentd/pull/4710/commits/8880cb679aa131c0692d43fe3bab232ce812cf06

**Review by daipom — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**